wfe/web-front-end_test.go

```diff
@@ -439,7 +439,7 @@ func TestIssueCertificate(t *testing.T) {
 	})
 	test.AssertEquals(t,
 		responseWriter.Body.String(),
-		"{\"type\":\"urn:acme:error:malformed\",\"detail\":\"Error unmarshaling certificate request\"}")
+		`{"type":"urn:acme:error:malformed","detail":"Unable to read/verify body"}`)
```

Comment:
No, this is testing the same thing.  The one below it is now testing for a body without a resource, and the one I added after that, with `'{"resource":"new-cert"}`, is the equivalent of the old `{}` test.